app/schemas/com.x8bit.bitwarden.data.vault.datasource.disk.database.VaultDatabase/4.json

```diff
@@ -253,4 +253,4 @@
       "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, 'f7906c69e0a2c065d4d3be140fc721b6')"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Is this going to mess up anything?

Reply:
Negative. Formatter is **really** insistent on having it there, so I conceded to the machine. 🤖

Reply:
Hmm, the other ones do not have. Neither does the new one.

To be clear, I like the newline. But we have been leaving them out here since these are autogenerated.